src/mlrose_ky/opt_probs/_opt_prob.py

```diff
@@ -230,19 +230,21 @@ def evaluate_population_fitness(self) -> None:
         """Evaluate the fitness of the current population."""
         self.pop_fitness = np.array([self.eval_fitness(indiv) for indiv in self.population])
 
-    def set_state(self, new_state: np.ndarray) -> None:
+    def set_state(self, new_state: np.ndarray, fitness: float) -> None:
```

Comment:
This new parameter was added without updating the test cases that cover `.set_state()`.